packages/math/src/apr.ts

```diff
@@ -24,18 +25,21 @@ const convertLockup = (lockup: string, durations: Duration[]) => {
   }
 };
 
-export const calcPoolAprs = ({
-  activeGauges,
-  pool,
-  prices,
-  superfluidPools,
-  aprSuperfluid,
-  lockupDurations,
-  volume7d,
-  swapFee,
-  lockup = "14",
-  includeNonPerpetual = true,
-}: CalcPoolAprsParams) => {
+export const calcPoolAprs = (
+  osmosisAssets: OsmosisAsset[],
```

Comment:
can we use `AssetList[]` from `@chain-registry/types`?

Reply:
> can we use `Asset` or `AssetList[]` from `@chain-registry/types`? or is has this been renamed?

yes we can use `Asset` directly, actually the `osmosisAsset` here is just an alias of `Asset`.
```
import { Asset as OsmosisAsset } from "@chain-registry/types";
```

so this means the math module is supposed to be a common library and could be used for all kinds of Asset, not only for osmosis, am I right?